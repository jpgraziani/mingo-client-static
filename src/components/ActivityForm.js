import React from 'react'

export default function ActivityForm(props) {
  const { className, ...otherProps } = props
  return (
    <form
      className={['Activity-Form', className].join(' ')}
      action='#'
      {...otherProps}
    />
  );
}