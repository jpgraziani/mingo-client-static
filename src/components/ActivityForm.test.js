import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ActivityForm from './ActivityForm'

describe(`ActivityForm component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  }

  it('renders a form.ActivityForm by default', () => {
    const wrapper = shallow(<ActivityForm />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the ActivityForm given props', () => {
    const wrapper = shallow(<ActivityForm {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})