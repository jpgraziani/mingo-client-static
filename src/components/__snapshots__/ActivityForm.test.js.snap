// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivityForm component renders a form.ActivityForm by default 1`] = `
<form
  action="#"
  className="Activity-Form "
/>
`;

exports[`ActivityForm component renders the ActivityForm given props 1`] = `
<form
  action="#"
  className="Activity-Form test-class-name"
  data-other="test-other-prop"
>
  <p>
    test children
  </p>
</form>
`;
